spec/features/newflow/educators/educator_signup_flow_spec.rb

```diff
@@ -0,0 +1,245 @@
+require 'rails_helper'
+
+module Newflow
+
+  feature 'Educator signup flow', js: true do
+
+    background { load 'db/seeds.rb' }
+    before(:each) { turn_on_educator_feature_flag }
+
+    let(:first_name) { "Elon#{Faker::Alphanumeric.alphanumeric(number: 20)}" }
```

Comment:
why not just use faker to generate the name instead of appending numbers?

Reply:
Good idea. Done.